vllm/plugins/__init__.py

```diff
@@ -25,6 +26,8 @@ def load_general_plugins():
     os.environ['TORCHINDUCTOR_COMPILE_THREADS'] = '1'
     # see https://github.com/vllm-project/vllm/issues/10619
     torch._inductor.config.compile_threads = 1
+    if current_platform.is_xpu():
+        os.environ['TORCH_COMPILE_DISABLE'] = 'True'
```

Comment:
any documentation for this env var?

Reply:
Do you mean where this config comes from? If so, please refer [here](https://github.com/pytorch/pytorch/blob/main/torch/_dynamo/config.py#L125).